apps/web/playwright/organization/organization-settings.e2e.ts

```diff
@@ -68,7 +68,18 @@ async function verifyRobotsMetaTag({ page, orgSlug, urls, expectedContent }: Ver
       const { url } = await goToUrlWithErrorHandling(relativeUrl);
       const metaTag = await page.locator('head > meta[name="robots"]');
       const metaTagValue = await metaTag.getAttribute("content");
-      expect(metaTagValue).toEqual(expectedContent);
+      expect(metaTagValue).not.toBeNull();
+      expect(
+        metaTagValue
+          ?.split(",")
+          .map((s) => s.trim())
+          .join(",")
+      ).toEqual(
+        expectedContent
+          .split(",")
+          .map((s) => s.trim())
+          .join(",")
+      );
```

Comment:
to prevent this test failure: https://github.com/calcom/cal.com/actions/runs/12771750003/job/35600401252?pr=18656